### PR TITLE
Added review to the docs

### PR DESCRIPTION
**Pull request by moujaune**
Added review to the docs

### CODE REVIEW

File: AlembicImporter/Packages/com.unity.formats.alembic/Documentation~/com.unity.formats.alembic.md [hunk 1]
```diff
@@ -103,7 +107,7 @@ The Timeline can be used to playback and record Alembic animation including:
 * Create sequences using Alembic clips with the ability to trim times and adjust clip-ins
 * Record Alembic data directly from the Timeline to an Alembic file
 
-
+> Consider making sure that you can actually edit Alembic Shot Clips via code or otherwise add it to known limitations
```
Comment (vkovec) [inline AlembicImporter/Packages/com.unity.formats.alembic/Documentation~/com.unity.formats.alembic.md]: where does it say  you can edit Alembic Shot Clips via code?

File: AlembicImporter/Packages/com.unity.formats.alembic/Documentation~/com.unity.formats.alembic.md [hunk 2]
```diff
@@ -78,7 +82,7 @@ To configure a scene to export an Alembic file, add the [Alembic Exporter compon
 The component can be configured to export the entire scene or individual object hierarchies.
 
 Using the AlembicExporter component automatically disables Draw Call Batching. If the Mesh group is valid after being batched then it will be exported. In some cases the data will be batched multiplied times and the results may change.  If you want to control the Batch settings they can be found in the Rendering section of Player Settings.
-
+> Consider clarifying section; maybe a reminder line about what batching does for people who mark everything as static in their scene for performance and panic when the exporter turns it off and the project starts to slow down
```
Comment (vkovec) [inline AlembicImporter/Packages/com.unity.formats.alembic/Documentation~/com.unity.formats.alembic.md]: please give an example of what you would add

Comment (moujaune) [reply AlembicImporter/Packages/com.unity.formats.alembic/Documentation~/com.unity.formats.alembic.md]: "Adding an AlembicExporter component to your scene will disable draw call batching in your project. With batching enabled, your scene's geometry merges itself in various configurations every frame, thus somewhat marring the faithfulness of the exporter's geometry recording features.

After you're done recording, you can re-enable both static and dynamic batching for your project in the Rendering section of the Player Settings. Having the exporter run with batching enabled will engender undesirable results such at batched meshes being recorded multiple times per frame and overlapping in your Alembic file in an unsightly fashion."

File: AlembicImporter/Packages/com.unity.formats.alembic/Documentation~/com.unity.formats.alembic.md (hunk 1 above)
Comment (vkovec) [reply AlembicImporter/Packages/com.unity.formats.alembic/Documentation~/com.unity.formats.alembic.md]: ah ok. We already mention that there is no external API though. Would that not cover this as well?

File: AlembicImporter/Packages/com.unity.formats.alembic/Documentation~/com.unity.formats.alembic.md (hunk 2 above)
Comment (vkovec) [reply AlembicImporter/Packages/com.unity.formats.alembic/Documentation~/com.unity.formats.alembic.md]: thanks!

**Pull request closed without merging**